src/ResourceManager/AnalysisServices/Commands.AnalysisServices/ChangeLog.md

```diff
@@ -20,7 +20,7 @@
 ## Current Release
 
 ## Version 0.6.8
-* Enable Gateway assocaite/disassociate operations on AS.
+* Enable Gateway assocaite/disassociate operations on AS. 
```

Comment:
revert this?

Reply:
Can't, it'll fail the checkchangelog script :(